pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -133,8 +133,10 @@ let
       # https://gitweb.gentoo.org/repo/gentoo.git/plain/www-client/chromium/
       # https://git.archlinux.org/svntogit/packages.git/tree/trunk?h=packages/chromium
       # for updated patches and hints about build flags
-      ++ optionals (versionRange "61" "62") [
+      ++ optionals (versionOlder version "62") [
       ./patches/chromium-gn-bootstrap-r14.patch
```

Comment:
I'm puzzling over how that should be different from the check included with versionRange: https://github.com/NixOS/nixpkgs/pull/29941/commits/8a5b327d790b3c6f12aa3f9754f3bdbbc817b5da#diff-f286bd0559a1a8a82983af7fe0830d69L95

Reply:
If there is an issue with `versionRange`, please fix that instead of circumventing it. It also functions to warn for outdated patch sets.